src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Boolean.cs

```diff
@@ -158,6 +163,110 @@ public override bool TryGetParamDescription(string paramName, out string paramDe
         }
     }
 
+    // Boolean(arg:b)
+    // Corresponding Excel and DAX function: Boolean
+    internal sealed class BooleanBFunction : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        public override bool SupportsParamCoercion => false;
+
+        public BooleanBFunction()
+            : base(BooleanFunction.BooleanInvariantFunctionName, TexlStrings.AboutBooleanN, FunctionCategories.Text, DType.Boolean, 0, 1, 1, DType.Boolean)
+        {
+        }
+
+        public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
+        {
+            yield return new[] { TexlStrings.BooleanArg1 };
+        }
+
+        public override string GetUniqueTexlRuntimeName(bool isPrefetching = false)
+        {
+            return GetUniqueTexlRuntimeName(suffix: "B");
```

Comment:
>return GetUniqueTexlRuntimeName(suffix: "B"); [](http://example.com/codeflow?start=12&length=45)

If this is not going to be implemented in JS (handled via codegen), we don't need this override. I'll work on removing this logic from PowerFx early next year.

Reply:
Same for next function

Reply:
removed.